app/main.py

```diff
@@ -293,24 +294,15 @@ def update_live_alerts_data(
     all_alerts = pd.DataFrame(response)
 
     # We now want to build the boolean indexing mask that indicates whether or not the event is unacknowledged
-    # We start by making an API call to fetch all events
-    response = api_client.get_past_events()
+    # We start by making an API call to fetch all unacknowledged events
+    response = api_client.get_unacknowledged_events()
     if response.status_code == 401:
         api_client.refresh_token(cfg.API_LOGIN, cfg.API_PWD)
-        response = api_client.get_past_events()
-    all_events = response.json()
-
-    # Then, we construct a dictionary whose keys are the event IDs (as integers) and values are the corresponding
-    # "is_acknowledged" field in the events table (boolean)
-    is_event_acknowledged = {}
-    for event in all_events:
-        is_event_acknowledged[event["id"]] = event["is_acknowledged"]
+        response = api_client.get_unacknowledged_events()
+    all_events = pd.DataFrame(response.json())
```

Comment:
not sure if that's on purpose, but those are not "all events", only the ones that haven't been acknowledged

Reply:
True but that's only a naming convention